src/config.yaml

```
@@ -31,7 +31,7 @@ params:
   header:
     project_shortname: "OSCAL"
     project_name: "OSCAL: the Open Security Controls Assessment Language"
-  page_repo_path: https://github.com/usnistgov/OSCAL/tree/main/docs/content
+  page_repo_path: https://github.com/usnistgov/OSCAL-Pages/tree/main/src/content
```

I wonder if the external contributors should push the proposed updates to main or to develop branch...

---

Good question but this only controls when they click what they see to edit, not how it gets saved. Once they choose to save that commit they have to choose to save to a branch for a PR, they will be unable to commit to branch, they will be forced to do so. GitHub's interface controls that, not me.

---

When you and Chris discussed this in review, I realized even if my answer stands your implied question about the target branch to change is worth calling out, I am updating my PR to point the `develop` branch not `main`.

```suggestion
  page_repo_path: https://github.com/usnistgov/OSCAL-Pages/tree/develop/src/content
```